### DISCUSSION TITLE
How To Assign Where the "New Note" Sidebar Button Inserts the New Note?

### DISCUSSION THREAD

**Discussion by mwsmedia** [TriliumNext/Trilium#3021] [Q&A]
Hi -- digging Trilium, but I'm noticing some functionality oddities.

Most frustrating: when clicking the "New Note" button on the left-hand sidebar, the new note automatically appears as a child of the Journal note for the current day -- even when other notes are selected as current.

Where is this controlled? I would prefer that the "New Note" button generates a new note either as a child of the currently selected note, or as a child of Root -- but ideally, I'd like this to be something the user can assign.

**Comment by FliegendeWurst** [comment]
You can create a new note as child of the selected note using the `createNoteInto` shortcut (default: Ctrl+P).

The "New Note" button is currently only set up to create a new child note of the "inbox" note ([source](https://github.com/zadam/trilium/blob/6c43b92bf1f39926728044e7bfb5c04f3f05b9fb/src/public/app/layouts/desktop_layout.js#L96-L99)). The inbox note is either the current day note or the note with the "inbox" label (if such a note exists).

**Reply by mwsmedia** [reply]
Thanks -- Is it possible to assign the New Note button to a different note without having to compile from source? Might be a good configuration setting to have.

**Reply by zadam** [reply]
I guess, there's some misunderstanding, there's no need to recompile from source. Trilium is partly configurable through [attributes](https://github.com/zadam/trilium/wiki/Attributes) (label is one type of them) and as @FliegendeWurst suggested, it is possible to achieve what you want by assigning label "inbox" to a note where you want your new notes to appear:

<img width="585" alt="image" src="https://user-images.githubusercontent.com/617641/181842426-662c19d6-68a7-498c-9233-892a48fabb4f.png">

**Reply by winamp8** [reply]
Solved my confusion!

**Comment by sigaloid** [comment]
@zadam : what do you think about separating into two buttons: new day note, or new child note?

![image](https://user-images.githubusercontent.com/69441971/181838362-bfeb0bac-e6ca-414e-b95f-8491d7b45d56.png)
![image](https://user-images.githubusercontent.com/69441971/181838385-65ed2b15-7068-47e3-87b4-d1896c94b513.png)

Only problem is there is no regular logo for the file with a plus button, only solid. I didn't want to make a new PR unless you think this is a worthy change.

**Reply by zadam** [reply]
The standard way to create a child is either through keyboard or through plus button on hover:

<img width="343" alt="image" src="https://user-images.githubusercontent.com/617641/181842640-992b37e2-10d8-4acd-8125-a33391c51f16.png">

I plan to make left sidebar configurable and at that point I think it will make sense to create more button combinations, but until then it would be sort of duplication which will take space for everyone even if you don't want to use it.
